cmd/calculaterelease/main.go

```diff
@@ -16,11 +16,24 @@ import (
 	"github.com/awslabs/aws-go-multi-module-repository-tools/release"
 )
 
+type preReleaseFlag string
+
+func (p *preReleaseFlag) String() string {
+	return string(*p)
+}
+
+func (p *preReleaseFlag) Set(s string) error {
+	*p = preReleaseFlag(s)
+	return nil
+}
+
+var preview preReleaseFlag
 var verbose bool
 var outputFile string
 
 func init() {
 	flag.BoolVar(&verbose, "v", false, "output with verbose changes")
+	flag.Var(&preview, "preview", "indicates a semver pre-release should be calculated for all modules.")
```

Comment:
Will the `preview` flag override the preview suffix for any existing module configured in `modman.toml`?  Or does this only apply to modules that are not preview?

Reply:
Applies to all modules, may be easier to sync offline on the reasoning for that.